programs/compressed-token/src/process_mint.rs

```diff
@@ -59,13 +67,11 @@ pub fn process_mint_to<'info>(
 
     #[cfg(target_os = "solana")]
     {
-        let inputs_len =
-    // struct
-    mem::size_of::<light_system_program::InstructionDataInvokeCpi>()
-    // `output_compressed_accounts`
-    + mem::size_of::<CompressedAccount>() * amounts.len()
-    // `output_state_merkle_tree_account_indices`
-    + amounts.len() + mem::size_of::<Option::<light_system_program::sdk::CompressedCpiContext>>()+ 8;
+        let inputs_len = 1 + 4 + 4 + 4 + amounts.len() * 170 + 1 + 1 + 1 + 26 + 1;
+        // inputs_len = Option<Proof> + Vec::new() + Vec::new() +
```

Comment:
I would prefer not hardcoded values, if that's possible. Any chance you could use `mem::size_of` on the types?

Reply:
mem size doesn't return the correct size because the structs have internal vectors

Reply:
replaced event sized for the same reason

Reply:
Gotcha, makes sense.